appveyor.yml

```diff
@@ -7,10 +7,13 @@ environment:
 
 services:
   - mssql2016
+  
+cache:
+  - C:\Users\appveyor\.m2 -> pom.xml
```

Comment:
I think it should be directory caching. By following snippet it will cache whole directory.

```
cache:
  directories:
    - $HOME/.m2
```
or
```
cache:
 - $HOME/.m2
```